ui/src/components/Publication/RelatedPublications/Modal/index.tsx

```diff
@@ -125,93 +121,93 @@ const RelatedPublicationsModal: React.FC<Props> = (props): React.ReactElement =>
-                                        >
-                                            Previous
-                                        </button>
+                                    Previous
+                                </button>
 
-                                        <button
-                                            onClick={() => {
-                                                setOffset((prev) => prev + limit);
-                                            }}
-                                            className="rounded font-semibold text-grey-800 underline decoration-teal-500 decoration-2 underline-offset-4 outline-none transition-colors duration-500 focus:ring-2 focus:ring-yellow-500 disabled:decoration-teal-600 disabled:opacity-70 dark:text-white-50"
-                                            disabled={limit + offset >= getCrosslinksResponse.metadata.total}
-                                        >
-                                            Next
-                                        </button>
-                                    </div>
-                                    <span className="mt-4 block font-medium text-grey-800 transition-colors duration-500 dark:text-white-50">
-                                        Showing {offset + 1} -{' '}
-                                        {limit + offset > getCrosslinksResponse.metadata.total
-                                            ? getCrosslinksResponse.metadata.total
-                                            : limit + offset}{' '}
-                                        of {getCrosslinksResponse.metadata.total}
-                                    </span>
-                                </Framer.motion.div>
-                            </Components.Delay>
-                        )}
-                    </>
-                )}
-            </Framer.AnimatePresence>
+                                <button
+                                    onClick={() => {
+                                        setOffset((prev) => prev + limit);
+                                    }}
+                                    className="rounded font-semibold text-grey-800 underline decoration-teal-500 decoration-2 underline-offset-4 outline-none transition-colors duration-500 focus:ring-2 focus:ring-yellow-500 disabled:decoration-teal-600 disabled:opacity-70 dark:text-white-50"
+                                    disabled={limit + offset >= getCrosslinksResponse.metadata.total}
+                                >
+                                    Next
+                                </button>
+                            </div>
+                            <span className="mt-4 block font-medium text-grey-800 transition-colors duration-500 dark:text-white-50">
+                                Showing {offset + 1} -{' '}
+                                {limit + offset > getCrosslinksResponse.metadata.total
```

Comment:
Could this block be put in a separate function? 

```
const getTotal = () => {
    if ((limit + offset) >  getCrosslinksResponse.metadata.total) {
        return getCrosslinksResponse.metadata.total;
    } else {
        return (limit + offset);
    }
};

...

<span className="mt-4 block font-medium text-grey-800 transition-colors duration-500 dark:text-white-50">
    Showing {offset + 1} -{' '}
    {getTotal()}
    {' '} of {getCrosslinksResponse.metadata.total}
</span>
```

Maybe, though still looks a bit messy. Think it's all these {' '} that's needed, 2024 and we still need to hardcode in spaces :( Remember the days of `&nbsp;` everywhere?

Reply:
Yeah, it is a bit messy as is. I decided to create a variable instead, I think it looks a bit neater now. See what you think